index.d.ts

```diff
@@ -67,6 +69,27 @@ export interface Options extends OperationOptions {
 	```
 	*/
 	readonly signal?: AbortSignal;
+
+	/**
+	Callback invoked on to determine whether a retry should occur. Receives the error thrown by `input` as the first argument with properties `attemptNumber` and `retriesLeft` which indicate the current attempt number and the number of attempts left, respectively.
+
+	`shouldRetry` will not allow AbortError overrides. It allows overriding the default retry logic (aborting on TypeError's that aren't network errors) for all other errors.
```

Comment:
> `shouldRetry` will not allow AbortError overrides.

I don't think it's clear enough what this means. I would spell it out more explicitly. That this cannot be used to prevent an aborted operation (AbortError).

Reply:
Got it! Will update this to make it clear on what this does and how it behaves.